apps/style/common.scss

```diff
@@ -1558,13 +1558,61 @@ $see-examples-thick-underline-color: $yellow;
   max-width: 100% !important;
 }
 
-a.WireframeSendToPhone_send-to-phone-link {
+.WireframeButtons_container {
+  display: flex;
```

Comment:
Does IE11 support the flexbox features we're using here?  Listed as partial support on http://caniuse.com/#feat=flexbox.